README.md

```diff
@@ -115,6 +115,18 @@ Our RCL is structured around the concepts put forth by [Atomic Design](http://br
 
 ## Contributing
 
+### Gotchas:
+
+#### SCSS / CSS and How our CSS-Module config works
+
+- Our CSS Module config creates complex, hashed classes like this: `animate-module__shake--3Q0r1` which is `[name of the component]__[local classname]--[hash:base64:5]'`.
+- When webpack comes across an imported SCSS file (`import styles from './button.module.scss'`), a new class is created name-spaced with the directory from where that file exists and added to the file css output file.
```

Comment:
`and added to the file css output file` - don't think we need that first `file`?